### DISCUSSION TITLE
Mastodon::UnexpectedResponseError: <Server URL>/inbox returned code 401 After Migrating to a new machine

### DISCUSSION THREAD

**Discussion by tmorio** [mastodon/mastodon#21404] [Q&A] [answered]
Mastodon Version: v3.5.5

I use translation because I can't speak English well.

I migrated the server using this page as a guide.
https://docs.joinmastodon.org/admin/migrating/

After the migration, I can log in to the server and post, but posts are not being pushed to other instances, and I cannot see posts from other instances on the timeline.

I checked Sidekiq and found the following error.
```
Mastodon::UnexpectedResponseError: https://<Server Domain>/inbox returned code 401
```

Are there any other points to be checked or operations to be performed to resolve the issue?

**Comment by tmorio** [comment, marked as answer]
This matter has been resolved.

Due to the DNS record cache, the other server was referencing the IP address of the old server.
Therefore, we have confirmed that as soon as the DNS cache is updated, the system will work properly.

**Reply by alisyd** [reply]
I am on my local machine,  behind a router with static ip, I have also migrated from old server to a new server, all behind the router, but I am still getting unexpected 401 error, on following users from other instances. Because I am behind a router, my Ip address on on the new machine is also same the one on my old machine, I am using the same domain and ssl on the new machine, but I am still getting 401 unexpected delivery errors.
